docs/src/docker.md

```diff
@@ -75,51 +89,46 @@ See our [Continuous Integration guides](./ci.md) for sample configs.
 
 ## Image tags
 
-See [all available image tags](https://mcr.microsoft.com/v2/playwright/tags/list).
+See [all available image tags].
```

Comment:
Was this intended to be removed? The link.
@yury-s

Reply:
Yes. It is now provided in pw.dev: https://github.com/microsoft/playwright.dev/commit/bf43194e5a88438f59f361967bb29f69a0cae7c4

Reply:
cool, thanks for confirming.